### PR TITLE
example code, as part of #88

### PR DESCRIPTION
**Pull request by mcr**

### CODE REVIEW

**Review by EskoDijk — CHANGES_REQUESTED** [review]
This will probably need to be integrated into the current Appendix that Peter added. See "{libsup}" to find it.  I prefer an Appendix as the location for this code but we can discuss in the call.

**Comment by petervanderstok** [comment]
Just what I wanted to write myself. I agree.
I am less sure about the actual code. It has a tendency to be ephemeral 
with changing libraries....

Peter

Esko Dijk schreef op 2021-04-29 09:18:

> @EskoDijk requested changes on this pull request.
> 
> This will probably need to be integrated into the current Appendix that 
> Peter added. See "{libsup}" to find it. I prefer an Appendix as the 
> location for this code but we can discuss in the call.
> 
> --
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub [1], or unsubscribe 
> [2].


Links:
------
[1] 
https://github.com/anima-wg/constrained-voucher/pull/113#pullrequestreview-647879742
[2]

File: constrained-voucher.mkd [hunk 1]
```diff
@@ -921,3 +921,90 @@ INSERT_FIG_FROM_FILE examples/voucher-from-MASA.txt END
...
+                    BIO_printf(bio_out, " (untrusted)");
+                BIO_printf(bio_out, "\n");
+            }
+            sk_X509_pop_free(chain, X509_free);
+        }
+    } else {
+        BIO_printf(bio_err,
+                   "error %s: verification failed\n",
+                   (file == NULL) ? "stdin" : file);
+    }
```
Comment (mcr) [inline constrained-voucher.mkd:990]:
```suggestion

```

**Comment by mcr** [comment]
Move up into appendix A, "libsub"

**Pull request merged**